test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Checking out the i64-store8 branch and compiling again I get something totally different:
```
(set_local $$rem
  (call $i64u-rem
    (get_local $$x$addr$012)
    (i64.const 10)
  )
)
(set_local $$add
  (i64.or
    (get_local $$rem)
    (i64.const 48)
  )
)
(set_local $$conv
  (i32.and
    (i32.wrap/i64
      (get_local $$add)
    )
    (i32.const 255)
  )
)
(set_local $$incdec$ptr
  (i32.add
    (get_local $$s$addr$013)
    (i32.const -1)
  )
)
(i32.store8
  (get_local $$incdec$ptr)
  (get_local $$conv)
)
```
which doesn't even look like the same style so I'm probably doing something wrong.

Are the i64.store8s being generated in `(func $_fmt_u`?